scripts/provisioning/s3_start

```diff
@@ -44,35 +36,57 @@ def main():
   as a argument.
 
   """
-  s3config_path = "/etc/cortx/s3/conf/s3config.yaml"
   parser = argparse.ArgumentParser("S3server single entry command")
   parser.add_argument("--service", required=True)
   parser.add_argument("--index", required=False)
+  parser.add_argument("-c", required=True)
 
   args = parser.parse_args()
 
-  create_logger_directory()
-  logger = create_logger()
-
-  # TODO: to be removed when end-to-end logging integration is done:
-  os.makedirs("/var/log/cortx/s3", mode=0o755, exist_ok=True)
-
   try:
+    if not args.c:
+      raise Exception("provide confstore_url")
+    confstore_url = args.c
+
+    # Load confstore file
+    Conf.load('confstore_index', confstore_url)
+
+    # Get Config entries
+    config_path = Conf.get('confstore_index', "cortx>common>storage>config")
+    logdir_path = Conf.get('confstore_index', "cortx>common>storage>log")
+
+    #Config path details
+    s3config_path = os.path.join(config_path, "s3", "conf", "s3config.yaml")
+    sysconfig_path = os.path.join(config_path, "s3", "sysconfig")
+
+    #Logging path details
+    s3_single_entry_log_directory = \
+        os.path.join(logdir_path, "s3", "s3deployment")
+    s3_single_entry_log_file = \
+        os.path.join(s3_single_entry_log_directory, "s3_single_entry.log")
+    s3_single_entry_logger_name = \
+        "s3-single_entry-logger-" + "[" + str(socket.gethostname()) + "]"
```

Comment:
Append machine ID
Get it from confstore

Reply:
Done.